src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Export/ExportJobTaskTests.cs

```diff
@@ -12,6 +12,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Hl7.Fhir.ElementModel;
+using MediatR;
```

Comment:
Can you add an assert to at least one of these tests that the Mediator substitute is being called. I saw the tests in the anonymizer test file, but since this isn't something specifically for anonymizer I would like to see it test here too.

Reply:
Thanks, updated.